components/Downloads/SecondaryDownloadMatrix.tsx

```diff
@@ -1,14 +1,16 @@
 import DownloadList from './DownloadList';
 import { useNextraContext } from '../../hooks/useNextraContext';
-import type { NodeVersionData, LegacyDownloadsFrontMatter } from '../../types';
+import type { LegacyDownloadsFrontMatter, NodeReleaseData } from '../../types';
 import type { FC } from 'react';
 
-type SecondaryDownloadMatrixProps = Pick<NodeVersionData, 'node'>;
+type SecondaryDownloadMatrixProps = {
+  release: NodeReleaseData;
+};
 
 // @TODO: Instead of using a static list it should be created dynamically. This is done on `nodejs.dev`
 // since this is a temporary solution and going to be fixed in the future.
 const SecondaryDownloadMatrix: FC<SecondaryDownloadMatrixProps> = ({
```

Comment:
```suggestion
const SecondaryDownloadMatrix: FC<NodeReleaseData> = ({
```

Same here, I think we can make the release data be directly done from here